tests/bootstrap_tests/test_resolver.py

```diff
@@ -933,13 +933,13 @@ def test_resolve_latest_cached_configuration(self):
         )
 
         config = self._resolver.resolve_configuration(
-            {"type": "app_store", "name": "latest_test", "version": "v0.1.1"},
+            {"type": "app_store", "name": "latest_test", "version": "v0.1.0"},
```

Comment:
Why did this suddenly change? Could you explain in the PR why? Doesn't seem from your changes that we shouldn't be able to resolve v0.1.1 anymore...